assembly/src/main/jib/app/stream-registry-config.yaml

```diff
@@ -9,6 +9,10 @@ server:
 # Env where StreamRegistry application is running
 env: ${STREAM_REGISTRY_ENVIRONMENT:-local}
```

Comment:
MDCFilters are parameterized so that different Customers can use different ones.